.github/workflows/rust.yml

```diff
@@ -101,7 +101,7 @@ jobs:
         env:
           RUSTFLAGS: '-Zinstrument-coverage'
           LLVM_PROFILE_FILE: '%p-%m.profraw'
-          EXTRA_CARGO_ARGS: '--verbose'
+          EXTRA_CARGO_ARGS: '--features std_fs'
```

Comment:
No no, I meant modifying Makefile. `test_matrix` is an entry dedicated for testing different feature combinations.

Reply:
Are you suggesting define a matrix like this: https://github.com/MichaelScofield/raft-engine/commit/61ccf8d14666cdbfb340a73ec67d6fbe4503525b ?

Reply:
No, add "cargo test --features nightly_group,failpoints,std_fs xxx" to https://github.com/tikv/raft-engine/blob/58041ad3e69ae76941309733bba936743bf8b5a5/Makefile#L67

No need to update rust.yml.

Reply:
You mean like this https://github.com/MichaelScofield/raft-engine/commit/bdc6d68860563827df26711dd269b36726e810b4 ?